go.mod

```diff
@@ -168,7 +168,7 @@ require (
 	go.uber.org/zap v1.14.0
 	golang.org/x/crypto v0.0.0-20210616213533-5ff15b29337e
 	golang.org/x/lint v0.0.0-20200130185559-910be7a94367
-	golang.org/x/net v0.0.0-20210226172049-e18ecbb05110
+	golang.org/x/net v0.0.0-20210614182718-04defd469f4e
```

Comment:
Why is this updated?

Reply:
don't know, maybe some kind of unresolved conflict ^^, seems to be good now